src/components/GuessAutocompleteList.vue

```diff
@@ -66,6 +66,7 @@ const vScrollIntoView: Directive<HTMLElement, boolean> = {
   align-items: stretch;
   max-height: calc((var(--count) + 0.6) * (1em + var(--v-padding) * 2));
   overflow-y: scroll;
+  z-index: 10;
```

Comment:
Fixes this issue, admittedly a bit haphazard since there is no `z-indexes.ts` yet, which we can add in this PR too.
![CleanShot 2025-05-20 at 22 46 42](https://github.com/user-attachments/assets/109029c5-504c-4cda-a343-0d346adb1c76)